verticapy/connection/connect.py

```diff
@@ -140,6 +181,22 @@ def current_connection() -> GlobalConnection:
     connect  using  an  auto  connection.  Otherwise,
     VerticaPy  attempts  to connect to a  VerticaLab
     Environment.
+
+    Example
+    -------
+    .. code-block:: python
+
+        from verticapy.connection import current_connection
```

Comment:
please add a space between the import and the code.
Add also a line:
Let's get the current VerticaPy connection. - (Please feel free to reformulate if it is not good)
Then another one:
Once the connection is established, you can execute SQL queries directly. 
.. note:: Please refer to your connector's API reference for a comprehensive list of its functionalities.